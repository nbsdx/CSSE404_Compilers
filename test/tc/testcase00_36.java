// invalid assignment

class A
{
   public static void main( String[] args )
   {
      int b = (new B()).start();
   }
}

class B
{
   public boolean start()
   {
      return true;
   }
}

