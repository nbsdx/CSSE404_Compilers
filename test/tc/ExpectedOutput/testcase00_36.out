Cannot assign type boolean to variable b of type int
